presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -58,7 +59,14 @@ public Connector create(String connectorId, Map<String, String> requiredConfig)
         requireNonNull(optionalConfig, "optionalConfig is null");
 
         try (ThreadContextClassLoader ignored = new ThreadContextClassLoader(classLoader)) {
-            Bootstrap app = new Bootstrap(new JdbcModule(connectorId), module);
+            Bootstrap app = new Bootstrap(
+                    new JdbcModule(connectorId),
+                    module,
+                    installModuleIf(
+                            JdbcSecurityConfig.class,
+                            security -> "none".equalsIgnoreCase(security.getPostgreSqlSecuritySystem()),
```

Comment:
That's a good point. I was debating between the two designs -- having the same set of access control checks for Postgres and Mysql versus keeping the two implementations different. I went with separate implementation for Postgres (and eventually Mysql) to provide the flexibility. Coming to think of it, we can have a base implementation which is same for both the connectors and then override it when it differs. 

Though in my opinion, Postgres and Mysql should each have their own configuration property (postgres.security and mysql.security) which is what the above piece of code is referring to.